tokens/README.md

```diff
@@ -125,3 +125,113 @@ recipient address. That SOL can be used to pay transaction fees on staking
 operations such as delegating stake. The rest of the allocation is put in
 a stake account. The new stake account address is output in the transaction
 log.
+
+## Distribute SPL tokens
+
+Distributing SPL Tokens works very similarly to distributing SOL, but requires
+the `--owner` parameter to sign transactions. Each recipient account must be an
+initialized SPL Token Account that shares the same mint as the sender in order
+for an allocation to succeed.
+
+Send SPL tokens to the recipients in `<RECIPIENTS_CSV>`.
+
+Example recipients.csv:
+
+```text
+recipient,amount,lockup_date
```

Comment:
`lockup_date` is just an artifact of using the same CVS format for all distributions?

Reply:
Yes, exactly. Removing that header when not needed is not impossible, but a bit of a pain, iirc. I'll give it another go-over.

Reply:
✅